common/test/acceptance/pages/lms/course_wiki.py

```diff
@@ -79,3 +97,41 @@ def save_wiki_content(self):
         """
         self.q(css='button[name="save"]').click()
         self.wait_for_element_presence('.alert-success', 'wait for the article to be saved')
+
+
+class CourseWikiChangesPage(CourseWikiSubviewPage):
+    """
+    Course wiki changes page.
+    """
+
+    def is_browser_on_page(self):
+        """
+        Return if the browser is on the history page.
```

Comment:
Yes, the label on the wiki is "Changes", but as you can see from the URL and classes, internally it's "history". I can make it all consistently history.